api/services/SiteCreator.js

```diff
@@ -38,6 +38,34 @@ function paramsForNewBuild({ user, site, template }) {
   };
 }
 
+function ownerIsFederalistUser(owner) {
+  return User.findOne({
+    where: { username: owner },
+    attributes: ['username'],
+  });
+}
+
+function checkGithubOrg({ user, owner }) {
+  return ownerIsFederalistUser(owner)
+  .then((model) => {
+    if (model) {
+      // Owner of the repo is a user, not an org.
+      // They exist in our DB, drop down to the next promise and bypass
+      return Promise.resolve(true);
+    }
+
+    return GitHub.checkOrganizations(user, owner);
+  })
+  .then((existingOrg) => {
+    if (!existingOrg) {
+      throw {
+        message: `Organization '${owner}'' hasn't approved access for federalist. Ask an owner to authorize it`,
```

Comment:
Few editorial suggestions:
- accidental double single quote: `'${owner}''`
- capitalize `Federalist`
- add `.` after `it` in second sentence